### PR TITLE
fix: make subcommand public and remove --beta required flag

### PR DESCRIPTION
**Pull request by blackboxsw**
## Proposed Commit Message

Since we won't release until `ua fix` is production ready,
we can remove the --beta param and allow this command to show
in ua --help.

## Test Steps
CI should cover this, just a revert of a previous accepted commit.

## Desired commit type::

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [*] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:

 - [*] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: uaclient/cli.py [hunk 1]
```diff
@@ -226,30 +226,25 @@ def attach_parser(parser):
...
-            " Format: CVE-yyyy-nnnn, CVE-yyyy-nnnnnnn or USN-nnnn-dd"
+            " Format: CVE-yyyy-nnnn or USN-nnnn"
         ),
     )
     return parser
 
 
 def action_fix(args, cfg, **kwargs):
     if not re.match(security.CVE_OR_USN_REGEX, args.security_issue):
+        # TODO(review of error messaging to reduce output)
```
Comment (mitechie) [inline uaclient/cli.py:245]: I've not reviewed how we handle todo in the codebase. I want to raise the question on it's cool to have a todo, but in the past I've liked projects that the todo referenced a bug/issue number that could provide more context, discussion, and progress indication.

File: uaclient/cli.py [hunk 2]
```diff
@@ -837,7 +832,7 @@ def get_parser():
         help="refresh Ubuntu Advantage services from contracts server",
     )
     parser_fix = subparsers.add_parser(
-        "fix"  # BETA omit help string to hide subcommand from help
+        "fix", help="fix CVE and USN security issues on this machine"
```
Comment (mitechie) [inline uaclient/cli.py:835]: what about 

    help="check for and mitigate the impact of a CVE/USN on this system"

**Review by mitechie — APPROVED** [review]
ty, one comment and one suggestion on help text.

File: uaclient/cli.py (hunk 1 above)
Comment (blackboxsw) [reply uaclient/cli.py:245]: Good suggestion here and we should make the TODO() reference an issue. Turns out, this commit wasn't a clean revert, I shouldn't have included that comment in this as Lucas had addressed that point when the former PR was in WIP status. I've reapplied the proper diff which included his review comments. So this comment goes away.

**Pull request merged**